scripts/pylib/display-twister-harness/camera_shield/README.rst

```diff
@@ -0,0 +1,124 @@
+==============
+Display capture Twister harness
+==============
+
+
+Configuration example
+---------------------
+
+```yaml: config.yaml
+case_config:
+  device_id: 0  # Try different camera indices
+  res_x: 1280   # x resolution
+  res_y: 720    # y resolution
+  fps: 30       # analysis frame pre-second
+  run_time: 20  # Run for 20 seconds
+tests:
+  timeout: 30   # second wait for prompt string
+  prompt: "screen starts" # prompt to show the test start
+  expect: ["tests.drivers.display.check.shield"]
+plugins:
+  - name: signature
+    module: plugins.signature_plugin
+    class: VideoSignaturePlugin
+    status: "enable"
+    config:
+      operations: "compare" # operation ('generate', 'compare')
+      metadata:
+        name: "tests.drivers.display.check.shield" # finger-print stored metadata
+        platform: "frdm_mcxn947"
+      directory: "./fingerprints" # fingerprints directory to compare with, not used in generate mode
+      duration: 100 # number of frames to check
+      method: "combined" #Signature method ('phash', 'dhash', 'histogram', 'combined')
+      threshold: 0.65
+      phash_weight: 0.35
+      dhash_weight: 0.25
+      histogram_weight: 0.2
+      edge_ratio_weight: 0.1
+      gradient_hist_weight: 0.1
+```
```

Comment:
As you can see here, this does not render well in RST syntax:
https://github.com/zephyrproject-rtos/zephyr/blob/f32242fd568e871b1c6b2a471e36a5f2cdb5ae81/scripts/pylib/display-twister-harness/camera_shield/README.rst#configuration-example

You might wish to use [`.. code-block::`](https://thomas-cokelaer.info/tutorials/sphinx/rest_syntax.html#code-block-directive)